db/seeds/index.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Locations = require('./data/locations');
+const Movies    = require('./data/movies');
 
 exports.seed = function (Knex) {
   return Knex('movies').truncate()
   .then(() => {
     return Knex('movies').insert(Movies);
+  })
+  .then(() => Knex('locations').truncate())
+  .then(() => {
+    return Knex('locations').insert(Locations);
```

Comment:
You can use [`Bluebird.all`](http://bluebirdjs.com/docs/api/promise.all.html) to make this a little easier.

```js
return Bluebird.all([
  Knex('movies').truncate();
  Knex('locations').truncate();
])
.then(() => {
  return Bluebird.all([
    Knex('movies').insert(Movies),
    Knex('locations').insert(Locations)
  ]);
})
```